src/features/auth/components/SSOButton/SSOButton.native.test.tsx

```diff
@@ -139,6 +130,47 @@ describe('<SSOButton />', () => {
 
     expect(analytics.logLogin).toHaveBeenCalledWith({ method: 'fromLogin', type: 'SSO_login' })
   })
+
+  describe('When shouldLogInfo remote config is false', () => {
+    beforeAll(() => {
+      useRemoteConfigContextSpy.mockReturnValue({
+        ...DEFAULT_REMOTE_CONFIG,
+        shouldLogInfo: false,
+      })
+    })
+
+    it('should not log to Sentry on SSO login error', async () => {
+      useFeatureFlagSpy.mockReturnValueOnce(true)
+      jest.spyOn(GoogleSignin, 'signIn').mockRejectedValueOnce('GoogleSignIn Error')
```

Comment:
Comme il n'y a qu'un seul test ici, est-ce que ça vaudrait pas le coup d'enlever le describe et tout mettre dans le it ?

Reply:
Est-ce que ça ne peut pas poser problème en cas de plusieurs re-rendu si je fais un `mockReturnValueOnce`?

Reply:
ah oui ça peut être pénible en effet... D'ailleurs je pense qu'il faut que tu reset le mock de useRemoteConfigContextSpy after all du coup ? je ne sais pas qu'elle est sa valeur par défaut

Reply:
J'ai ajouté le `afterAll` que sur les `beforeAll` où je passais le remote config à true car false est sa valeur par défaut